pro/src/routes/routes_map.tsx

```diff
@@ -287,6 +289,18 @@ const routes: IRoute[] = [
     path: '/offres/collectives',
     title: 'Offres',
   },
+  {
+    component: CollectiveOfferConfirmation,
+    exact: true,
+    path: '/offre/:offerId([A-Z0-9]+)/collectif/confirmation',
+    title: 'Page de confirmation de création d’offre',
+  },
+  {
+    component: CollectiveOfferConfirmation,
+    exact: true,
+    path: '/offre/:offerId(T-[A-Z0-9]+)/collectif/confirmation',
+    title: 'Page de confirmation de création d’offre',
+  },
```

Comment:
react-router-v6 refuse les regex, autant eviter de les utiliser dans les nouvelles routes.
```suggestion
{
    component: CollectiveOfferConfirmation,
    exact: true,
    path: '/offre/:offerId/collectif/confirmation',
    title: 'Page de confirmation de création d’offre',
  },
```

Reply:
Ca risque de poser problème sur le collectif 😬 Enfin on devra faire des wrappers, tu sais quand est ce que la migration est prévue?